controllers/core/openstackversion_controller.go

```diff
@@ -301,6 +301,15 @@ func (r *OpenStackVersionReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		Log.Info("Setting DeployedVersion")
 		instance.Status.DeployedVersion = &instance.Spec.TargetVersion
 	}
+	if instance.Status.AvailableVersion != nil &&
```

Comment:
I don't think it is possible for AvailableVersion to be nil here.

Reply:
done